Source/ACE.Database/ShardDatabase.cs

```diff
@@ -293,7 +293,7 @@ public uint GetNextCharacterId()
         {
             uint maxId = ExecuteConstructedGetAggregateStatement<ShardPreparedStatement, CachedCharacter, uint>(ShardPreparedStatement.GetNextCharacterId);
 
-            ObjectGuid nextGuid = new ObjectGuid(maxId + 1, GuidType.Player);
+            ObjectGuid nextGuid = new ObjectGuid(maxId + 1);
```

Comment:
Since you changed to using the GuidManager for PlayerId's this function should be deleted entirely.